sway_libs/src/nft/SPECIFICATION.md

```diff
@@ -17,7 +17,7 @@ Table of Contents
             - [`set_admin()`](#set-admin)
         - [Burnable](#burnable)
             - [`burn()`](#burn)
-        - [Metadata](#metadata)
+        - [Token Metadata](#metadata)

```

Comment:
Some of the links in the ToC do not work given the renaming. I suggest going through the entire table to see if all of the links take you to the relevant section.